foundationdb/src/main/java/site/ycsb/db/foundationdb/FoundationDBClient.java

```diff
@@ -63,13 +65,15 @@ public void init() throws DBException {
     String clusterFile = props.getProperty(CLUSTER_FILE, CLUSTER_FILE_DEFAULT);
     String dbBatchSize = props.getProperty(DB_BATCH_SIZE, DB_BATCH_SIZE_DEFAULT);
     dbName = props.getProperty(DB_NAME, DB_NAME_DEFAULT);
+    String datacenterId = props.getProperty(DATACENTER_ID, DATACENTER_ID_DEFAULT);
 
     logger.info("API Version: {}", apiVersion);
     logger.info("Cluster File: {}\n", clusterFile);
 
     try {
       fdb = FDB.selectAPIVersion(Integer.parseInt(apiVersion.trim()));
       db = fdb.open(clusterFile);
+      db.options().setDatacenterId(datacenterId);
```

Comment:
We should default datacenterId to "", and change this to:
```java
if (!datacenterId.equals("")) {
//...
}
```
so this doesn't cause trouble for non-HA setups.